examples/workshop-r-shiny-styling/.saturn/saturn.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "workshop-r-shiny-styling",
+  "image_uri": "public.ecr.aws/saturncloud/saturn-rstudio:2022.04.01",
+  "description": "Supporting materials for the Styling Shiny workshop",
+  "working_directory": "/home/jovyan/examples/examples/workshop-r-shiny-styling",
+  "extra_packages": {
+    "cran": {
+      "install": "shiny"
```

Comment:
I think this is the only thing you'll need to change when you switch to "2022.01.06"